app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  has_many :likes, foreign_key: :author_id, dependent: :destroy
+  has_many :comments, foreign_key: :author_id, dependent: :destroy
+  has_many :posts, foreign_key: :author_id, dependent: :destroy
+
+  scope :user_recent_posts, -> { find(2).posts.order(created_at: :desc).first(3) }
```

Comment:
- [ ] Kindly do not tie the recent posts to a second user only. we can get our recent posts by removing the find method.

__HINT:__
```ruby
scope :user_recent_posts, -> { posts.order(created_at: :desc).first(3) }
```